Modules/Core/Common/include/itkFixedArray.h

```diff
@@ -262,8 +262,8 @@ class ITK_TEMPLATE_EXPORT FixedArray
 #    pragma GCC diagnostic ignored "-Warray-bounds"
 #  endif
 #endif
-  reference       operator[](unsigned int index) { return m_InternalArray[index]; }
-  const_reference operator[](unsigned int index) const { return m_InternalArray[index]; }
+  constexpr reference       operator[](unsigned int index) { return m_InternalArray[index]; }
```

Comment:
Adding `constexpr` here is a bit confusing. But I guess required to make `Modules/Core/Common/test/itkFixedArrayGTest.cxx:235` possible at compile time.

Reply:
@dzenanz Indeed, the constexpr `operator[]` is used here, in order to test the result of `FixedArray::Filled(value)` at compile time.

It doesn't seem bad to declare `operator[]`  as well, as C++17 also does so for `std::array::operator[]`: https://en.cppreference.com/w/cpp/container/array/operator_at  😃